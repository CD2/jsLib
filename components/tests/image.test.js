/* eslint-disable react/jsx-filename-extension */
import React from 'react'
import { Image } from '../Image'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

const wrapper = shallow(<Image />)
it(`snapshot`, ()=>expect(toJson(wrapper)).toMatchSnapshot())
