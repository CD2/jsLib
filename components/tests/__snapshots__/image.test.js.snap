// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <div
    className="accordion__header"
    onClick={[Function]}
  />
  <div
    className="accordion__content "
  />
</div>
`;
